_config/config.yml

```diff
@@ -21,3 +21,10 @@ SilverStripe\Security\Security:
 SilverStripe\MFA\Extension\AccountReset\SecurityExtension:
   extensions:
     mfaResetExtension: SilverStripe\MFA\Extension\AccountReset\MFAResetExtension
+---
+name: mfa-devconfig
+Only:
+  envvarset: 'BYPASS_MFA'
```

Comment:
Ideally we'd have a `envvarvalue` directive in the config layer, because from my understanding this would also kick in with `BYPASS_MFA=0` - but that's not really an issue this PR can address. See https://docs.silverstripe.org/en/4/developer_guides/configuration/configuration/